frontend/src/lib/components/runs/RunsTable.svelte

```diff
@@ -57,44 +58,102 @@
 	}
 
 	$: groupedJobs = groupJobsByDay(jobs.slice(0, nbOfJobs))
+
+	let flatJobs: Array<
+		| {
+				type: 'date'
+				date: string
+		  }
+		| {
+				type: 'job'
+				job: Job
+		  }
+	> = []
+
+	$: {
+		flatJobs = []
+		for (const [date, jobsByDay] of Object.entries(groupedJobs)) {
+			flatJobs.push({ type: 'date', date })
+			for (const job of jobsByDay) {
+				flatJobs.push({ type: 'job', job })
+			}
+		}
+	}
```

Comment:
write separate function and triggers otherwise it's super hard to read the dependency